README.md

```diff
@@ -85,7 +85,7 @@ Note that `PYTHONPATH` must be set to include the `timemachine` repo root. To ru
 For example, starting from a clean environment with the openeye license file in `~/.openeye`:
 
 ```shell
-OE_DIR=~/.openeye PYTHONPATH=. pytest -xsv tests/
+OE_DIR=~/.openeye PYTHONPATH=$PWD:$PYTHONPATH pytest -xsv tests/
```

Comment:
Thinking about it some more, we should probably remove the reference to `PYTHONPATH`. Users only get `pytest` and other test requirements from doing `pip install .[test]` and not from `pip install requirements`, so it should already be in their path.

Reply:
Agreed; I think the readme needs some general cleanup. I'm inclined to leave this for now since we mention `make build` as an alternative to `pip install .`, and the user could conceivably install `pytest` separately. Should we open an issue for a general readme cleanup?

Reply:
Hmm, actually, reading through it again, we do currently bless `pip install .` as the way to install. So I think it makes sense to just remove the `PYTHONPATH` setting as part of this PR.

Reply:
Removed in https://github.com/proteneer/timemachine/pull/955/commits/0e49d13adba83751f85392c5749efc8ad706ad7d